packages/xo-web/src/common/select-license.js

```diff
@@ -6,24 +6,61 @@ import { injectIntl } from 'react-intl'
 import { injectState, provideState } from 'reaclette'
 import { map } from 'lodash'
 
+import { renderXoItemFromId } from './render-xo-item'
+
+const License = ({ license, formatTime }) =>
+  _(
+    'expiresOn',
+    {
+      date:
+        license.expires !== undefined
+          ? formatTime(license.expires, {
+              day: 'numeric',
+              month: 'numeric',
+              year: 'numeric',
+            })
+          : '',
+    },
+    message => (
+      <option value={license.id}>
+        <span>
+          {license.id.slice(-4)} (
+          {license.boundObjectId
+            ? renderXoItemFromId(license.boundObjectId)
+            : license.expires !== undefined
+            ? message
+            : ''}
+          )
```

Comment:
IMO, displaying both information risks overloading the readability of the `Select`.

Reply:
True, but it may be useful/necessary. You may want to choose the license based on the expiration date and the fact that the license is already bound to a host doesn't make that need disappear. As long as the whole text is visible, I'd say we can put both.